---
layout: paper
title: "Systematic genome-wide annotation of spliceosomal proteins reveals differential gene family expansion."
image: /assets/images/papers/16344558.jpg
authors: Nuno L Barbosa-Morais, Maria Carmo-Fonseca, Samuel Apar√≠cio
ref: Barbosa-Morais et al. 2006. Genome Res.
journal: "Genome research <b>16</b>, 66-77 (2006)"
pdf: /assets/pdfs/papers/16344558.pdf
doi: 10.1101/gr.3936206
abbrev: "Genome Res <b>16</b>, 66-77 (2006)"
pub_year: 2006
---

<br />
<div data-badge-popover="right" data-badge-type="donut" data-pmid="16344558" data-hide-no-mentions="true" class="altmetric-embed"></div>

# Abstract

Although more than 200 human spliceosomal and splicing-associated proteins are known, the evolution of the splicing machinery has not been studied extensively. The recent near-complete sequencing and annotation of distant vertebrate and chordate genomes provides the opportunity for an exhaustive comparative analysis of splicing factors across eukaryotes. We describe here our semiautomated computational pipeline to identify and annotate splicing factors in representative species of eukaryotes. We focused on protein families whose role in splicing is confirmed by experimental evidence. We visually inspected 1894 proteins and manually curated 224 of them. Our analysis shows a general conservation of the core spliceosomal proteins across the eukaryotic lineage, contrasting with selective expansions of protein families known to play a role in the regulation of splicing, most notably of SR proteins in metazoans and of heterogeneous nuclear ribonucleoproteins (hnRNP) in vertebrates. We also observed vertebrate-specific expansion of the CLK and SRPK kinases (which phosphorylate SR proteins), and the CUG-BP/CELF family of splicing regulators. Furthermore, we report several intronless genes amongst splicing proteins in mammals, suggesting that retrotransposition contributed to the complexity of the mammalian splicing apparatus.

