---
layout: paper
title: "PyClone: statistical inference of clonal population structure in cancer."
image: /assets/images/papers/24633410.jpg
authors: Andrew Roth, Jaswinder Khattra, Damian Yap, Adrian Wan, Emma Laks, Justina Biele, Gavin Ha, Samuel Aparicio, Alexandre Bouchard-Côté, Sohrab P Shah
ref: Roth et al. 2014. Nat Methods.
journal: "Nature methods <b>11</b>, 396-8 (2014)"
pdf: /assets/pdfs/papers/24633410.pdf
doi: 10.1038/nmeth.2883
abbrev: "Nat Methods <b>11</b>, 396-8 (2014)"
pub_year: 2014
---

<br />
<div data-badge-popover="right" data-badge-type="donut" data-pmid="24633410" data-hide-no-mentions="true" class="altmetric-embed"></div>

# Abstract

We introduce PyClone, a statistical model for inference of clonal population structures in cancers. PyClone is a Bayesian clustering method for grouping sets of deeply sequenced somatic mutations into putative clonal clusters while estimating their cellular prevalences and accounting for allelic imbalances introduced by segmental copy-number changes and normal-cell contamination. Single-cell sequencing validation demonstrates PyClone's accuracy. 

