---
layout: paper
title: "Direct Transcriptional Consequences of Somatic Mutation in Breast Cancer."
image: /assets/images/papers/27498871.jpg
authors: Adam Shlien, Keiran Raine, Fabio Fuligni, Roland Arnold, Serena Nik-Zainal, Serge Dronov, Lira Mamanova, Andrej Rosic, Young Seok Ju, Susanna L Cooke, Manasa Ramakrishna, Elli Papaemmanuil, Helen R Davies, Patrick S Tarpey, Peter Van Loo, David C Wedge, David R Jones, Sancha Martin, John Marshall, Elizabeth Anderson, Claire Hardy,  , Violetta Barbashina, Samuel A J R Aparicio, Torill Sauer, Øystein Garred, Anne Vincent-Salomon, Odette Mariani, Sandrine Boyault, Aquila Fatima, Anita Langerød, Åke Borg, Gilles Thomas, Andrea L Richardson, Anne-Lise Børresen-Dale, Kornelia Polyak, Michael R Stratton, Peter J Campbell
ref: Shlien et al. 2016. Cell Rep.
journal: "Cell reports <b>16</b>, 2032-46 (2016)"
pdf: /assets/pdfs/papers/27498871.pdf
doi: 10.1016/j.celrep.2016.07.028
abbrev: "Cell Rep <b>16</b>, 2032-46 (2016)"
pub_year: 2016
---

<br />
<div data-badge-popover="right" data-badge-type="donut" data-pmid="27498871" data-hide-no-mentions="true" class="altmetric-embed"></div>

# Abstract

Disordered transcriptomes of cancer encompass direct effects of somatic mutation on transcription, coordinated secondary pathway alterations, and increased transcriptional noise. To catalog the rules governing how somatic mutation exerts direct transcriptional effects, we developed an exhaustive pipeline for analyzing RNA sequencing data, which we integrated with whole genomes from 23 breast cancers. Using X-inactivation analyses, we found that cancer cells are more transcriptionally active than intermixed stromal cells. This is especially true in estrogen receptor (ER)-negative tumors. Overall, 59% of substitutions were expressed. Nonsense mutations showed lower expression levels than expected, with patterns characteristic of nonsense-mediated decay. 14% of 4,234 rearrangements caused transcriptional abnormalities, including exon skips, exon reusage, fusions, and premature polyadenylation. We found productive, stable transcription from sense-to-antisense gene fusions and gene-to-intergenic rearrangements, suggesting that these mutation classes drive more transcriptional disruption than previously suspected. Systematic integration of transcriptome with genome data reveals the rules by which transcriptional machinery interprets somatic mutation.

