---
layout: paper
title: "Interpretable dimensionality reduction of single cell transcriptome data with deep generative models."
image: /assets/images/papers/29784946.png
authors: Jiarui Ding, Anne Condon, Sohrab P Shah
ref: Ding et al. 2018. Nat Commun.
journal: "Nature communications <b>9</b>, 2002 (2018)"
pdf: /assets/pdfs/papers/29784946.pdf
doi: 10.1038/s41467-018-04368-5
abbrev: "Nat Commun <b>9</b>, 2002 (2018)"
pub_year: 2018
---

<br />
<div data-badge-popover="right" data-badge-type="donut" data-pmid="29784946" data-hide-no-mentions="true" class="altmetric-embed"></div>

# Abstract

Single-cell RNA-sequencing has great potential to discover cell types, identify cell states, trace development lineages, and reconstruct the spatial organization of cells. However, dimension reduction to interpret structure in single-cell sequencing data remains a challenge. Existing algorithms are either not able to uncover the clustering structures in the data or lose global information such as groups of clusters that are close to each other. We present a robust statistical model, scvis, to capture and visualize the low-dimensional structures in single-cell gene expression data. Simulation results demonstrate that low-dimensional representations learned by scvis preserve both the local and global neighbor structures in the data. In addition, scvis is robust to the number of data points and learns a probabilistic parametric mapping function to add new data points to an existing embedding. We then use scvis to analyze four single-cell RNA-sequencing datasets, exemplifying interpretable two-dimensional representations of the high-dimensional single-cell RNA-sequencing data.

