---
layout: paper
title: "High-Resolution Single-Cell DNA Methylation Measurements Reveal Epigenetically Distinct Hematopoietic Stem Cell Subpopulations."
image: /assets/images/papers/30078558.jpg
authors: Tony Hui, Qi Cao, Joanna Wegrzyn-Woltosz, Kieran O'Neill, Colin A Hammond, David J H F Knapp, Emma Laks, Michelle Moksa, Samuel Aparicio, Connie J Eaves, Aly Karsan, Martin Hirst
ref: Hui et al. 2018. Stem Cell Reports.
journal: "Stem cell reports <b>11</b>, 578-592 (2018)"
pdf: /assets/pdfs/papers/30078558.pdf
doi: 10.1016/j.stemcr.2018.07.003
abbrev: "Stem Cell Reports <b>11</b>, 578-592 (2018)"
pub_year: 2018
---

<br />
<div data-badge-popover="right" data-badge-type="donut" data-pmid="30078558" data-hide-no-mentions="true" class="altmetric-embed"></div>

# Abstract

Increasing evidence of functional and transcriptional heterogeneity in phenotypically similar cells examined individually has prompted interest in obtaining parallel methylome data. We describe the development and application of such a protocol to index-sorted murineÂ and human hematopoietic cells that are highly enriched in their content of functionally defined stem cells. Utilizing an optimized single-cell bisulfite sequencing protocol, we obtained quantitative DNA methylation measurements of up to 5.7 million CpGs in single hematopoietic cells. In parallel, we developed an analytical strategy (PDclust) to define single-cell DNA methylation states through pairwise comparisons of single-CpG methylation measurements. PDclust revealed that a single-cell epigenetic state can be described by a small (<1%) stochastically sampled fraction of CpGs and that these states are reflective of cell identity and state. Using relationships revealed by PDclust, we derive near complete methylomes for epigenetically distinct subpopulations of hematopoietic cells enriched for functional stem cell content.

