---
layout: paper
title: "Probabilistic cell-type assignment of single-cell RNA-seq for tumor microenvironment profiling."
image: /assets/images/papers/31501550.jpg
authors: Allen W Zhang, Ciara O'Flanagan, Elizabeth A Chavez, Jamie L P Lim, Nicholas Ceglia, Andrew McPherson, Matt Wiens, Pascale Walters, Tim Chan, Brittany Hewitson, Daniel Lai, Anja Mottok, Clementine Sarkozy, Lauren Chong, Tomohiro Aoki, Xuehai Wang, Andrew P Weng, Jessica N McAlpine, Samuel Aparicio, Christian Steidl, Kieran R Campbell, Sohrab P Shah
ref: Zhang et al. 2019. Nat Methods.
journal: "Nature methods <b>16</b>, 1007-1015 (2019)"
pdf: /assets/pdfs/papers/31501550.pdf
doi: 10.1038/s41592-019-0529-1
abbrev: "Nat Methods <b>16</b>, 1007-1015 (2019)"
pub_year: 2019
---

<br />
<div data-badge-popover="right" data-badge-type="donut" data-pmid="31501550" data-hide-no-mentions="true" class="altmetric-embed"></div>

# Abstract

Single-cell RNA sequencing has enabled the decomposition of complex tissues into functionally distinct cell types. Often, investigators wish to assign cells to cell types through unsupervised clustering followed by manual annotation or via 'mapping' to existing data. However, manual interpretation scales poorly to large datasets, mapping approaches require purified or pre-annotated data and both are prone to batch effects. To overcome these issues, we present CellAssign, a probabilistic model that leverages prior knowledge of cell-type marker genes to annotate single-cell RNA sequencing data into predefined or de novo cell types. CellAssign automates the process of assigning cells in a highly scalable manner across large datasets while controlling for batch and sample effects. We demonstrate the advantages of CellAssign through extensive simulations and analysis of tumor microenvironment composition in high-grade serous ovarian cancer and follicular lymphoma.

