---
layout: paper
title: "clonealign: statistical integration of independent single-cell RNA and DNA sequencing data from human cancers."
image: /assets/images/papers/30866997.jpg
authors: Kieran R Campbell, Adi Steif, Emma Laks, Hans Zahn, Daniel Lai, Andrew McPherson, Hossein Farahani, Farhia Kabeer, Ciara O'Flanagan, Justina Biele, Jazmine Brimhall, Beixi Wang, Pascale Walters,  , Alexandre Bouchard-Côté, Samuel Aparicio, Sohrab P Shah
ref: Campbell et al. 2019. Genome Biol.
journal: "Genome biology <b>20</b>, 54 (2019)"
pdf: /assets/pdfs/papers/30866997.pdf
doi: 10.1186/s13059-019-1645-z
abbrev: "Genome Biol <b>20</b>, 54 (2019)"
pub_year: 2019
---

<br />
<div data-badge-popover="right" data-badge-type="donut" data-pmid="30866997" data-hide-no-mentions="true" class="altmetric-embed"></div>

# Abstract

Measuring gene expression of tumor clones at single-cell resolution links functional consequences to somatic alterations. Without scalable methods to simultaneously assay DNA and RNA from the same single cell, parallel single-cell DNA and RNA measurements from independent cell populations must be mapped for genome-transcriptome association. We present clonealign, which assigns gene expression states to cancer clones using single-cell RNA and DNA sequencing independently sampled from a heterogeneous population. We apply clonealign to triple-negative breast cancer patient-derived xenografts and high-grade serous ovarian cancer cell lines and discover clone-specific dysregulated biological pathways not visible using either sequencing method alone.

