---
layout: paper
title: "Epiclomal: Probabilistic clustering of sparse single-cell DNA methylation data."
image: /assets/images/papers/32966276.jpg
authors: Camila P E de Souza, Mirela Andronescu, Tehmina Masud, Farhia Kabeer, Justina Biele, Emma Laks, Daniel Lai, Patricia Ye, Jazmine Brimhall, Beixi Wang, Edmund Su, Tony Hui, Qi Cao, Marcus Wong, Michelle Moksa, Richard A Moore, Martin Hirst, Samuel Aparicio, Sohrab P Shah
ref: P E de Souza et al. 2020. PLoS Comput Biol.
journal: "PLoS computational biology <b>16</b>, e1008270 (2020)"
pdf: /assets/pdfs/papers/32966276.pdf
doi: 10.1371/journal.pcbi.1008270
abbrev: "PLoS Comput Biol <b>16</b>, e1008270 (2020)"
pub_year: 2020
---

<br />
<div data-badge-popover="right" data-badge-type="donut" data-pmid="32966276" data-hide-no-mentions="true" class="altmetric-embed"></div>

# Abstract

We present Epiclomal, a probabilistic clustering method arising from a hierarchical mixture model to simultaneously cluster sparse single-cell DNA methylation data and impute missing values. Using synthetic and published single-cell CpG datasets, we show that Epiclomal outperforms non-probabilistic methods and can handle the inherent missing data characteristic that dominates single-cell CpG genome sequences. Using newly generated single-cell 5mCpG sequencing data, we show that Epiclomal discovers sub-clonal methylation patterns in aneuploid tumour genomes, thus defining epiclones that can match or transcend copy number-determined clonal lineages and opening up an important form of clonal analysis in cancer. Epiclomal is written in R and Python and is available at https://github.com/shahcompbio/Epiclomal.

