---
layout: paper
title: "Retrospective evaluation of whole exome and genome mutation calls in 746 cancer samples."
image: /assets/images/papers/32958763.jpg
authors: Matthew H Bailey, William U Meyerson, Lewis Jonathan Dursi, Liang-Bo Wang, Guanlan Dong, Wen-Wei Liang, Amila Weerasinghe, Shantao Li, Yize Li, Sean Kelso,   Gordon Saksena, Kyle Ellrott, Michael C Wendl, David A Wheeler, Gad Getz, Jared T Simpson, Mark B Gerstein, Li Ding,  
ref: Bailey et al. 2020. Nat Commun.
journal: "Nature communications <b>11</b>, 4748 (2020)"
doi: 10.1038/s41467-020-18151-y
abbrev: "Nat Commun <b>11</b>, 4748 (2020)"
pub_year: 2020
---

<br />
<div data-badge-popover="right" data-badge-type="donut" data-pmid="32958763" data-hide-no-mentions="true" class="altmetric-embed"></div>

# Abstract

The Cancer Genome Atlas (TCGA) and International Cancer Genome Consortium (ICGC) curated consensus somatic mutation calls using whole exome sequencing (WES) and whole genome sequencing (WGS), respectively. Here, as part of the ICGC/TCGA Pan-Cancer Analysis of Whole Genomes (PCAWG) Consortium, which aggregated whole genome sequencing data from 2,658 cancers across 38 tumour types, we compare WES and WGS side-by-side from 746 TCGA samples, finding that ~80% of mutations overlap in covered exonic regions. We estimate that low variant allele fraction (VAF < 15%) and clonal heterogeneity contribute up to 68% of private WGS mutations and 71% of private WES mutations. We observe that ~30% of private WGS mutations trace to mutations identified by a single variant caller in WES consensus efforts. WGS captures both ~50% more variation in exonic regions and un-observed mutations in loci with variable GC-content. Together, our analysis highlights technological divergences between two reproducible somatic variant detection efforts.

